setup.py

```diff
@@ -102,7 +102,7 @@
       scripts=glob('scripts/*'),
       # making sure that numpy is installed before biom
       setup_requires=['numpy', 'cython'],
-      install_requires=['psycopg2', 'click', 'bcrypt', 'pandas',
+      install_requires=['psycopg2', 'click', 'bcrypt', 'pandas<2.0',
```

Comment:
Why the pin?

Reply:
Because pandas>2.0 brakes the tests. I saw that this was failing: https://github.com/qiita-spots/qiita/actions/runs/4940536571/jobs/8832562245 so installed the latests pandas version in my local computer and got a lot of errors; then installed 'pandas<2.0' and everything worked fine again.

Reply:
Okay, makes sense to punt. Some of the types of changes needed are outlined here https://github.com/biocore/scikit-bio/pull/1851

Reply:
Added https://github.com/qiita-spots/qiita/issues/3289 so we don't forget.